cmd/machine-config-operator/bootstrap.go

```diff
@@ -24,28 +24,34 @@ var (
 	}
 
 	bootstrapOpts struct {
-		etcdCAFile          string
-		rootCAFile          string
-		kubeCAFile          string
-		pullSecretFile      string
-		configFile          string
-		oscontentImage      string
-		infraConfigFile     string
-		networkConfigFile   string
-		imagesConfigMapFile string
-		mccImage            string
-		mcsImage            string
-		mcdImage            string
-		etcdImage           string
-		setupEtcdEnvImage   string
-		infraImage          string
-		destinationDir      string
+		etcdCAFile                string
+		rootCAFile                string
+		kubeCAFile                string
+		pullSecretFile            string
+		configFile                string
+		oscontentImage            string
+		infraConfigFile           string
+		networkConfigFile         string
+		imagesConfigMapFile       string
+		mccImage                  string
+		mcsImage                  string
+		mcdImage                  string
+		etcdImage                 string
+		setupEtcdEnvImage         string
+		infraImage                string
+		destinationDir            string
+		etcdMetricsCAFile         string
+		etcdMetricsServerCertFile string
+		etcdMetricsServerKeyFile  string
 	}
 )
 
 func init() {
 	rootCmd.AddCommand(bootstrapCmd)
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.etcdCAFile, "etcd-ca", "/etc/ssl/etcd/ca.crt", "path to etcd CA certificate")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.etcdMetricsCAFile, "etcd-metrics-ca", "/assets/tls/etcd-metrics-ca-bundle.crt", "path to etcd metrics CA certificate")
```

Comment:
are these files actually there on bootstrap? meaning, the installer is already laying down these files right?

Reply:
No but I am injecting this with my own installer and using this release. https://github.com/openshift/installer/pull/1389

Reply:
It still fails

Reply:
```
OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=registry.svc.ci.openshift.org/ci-op-wz3pbz49/release:latest openshift-install-dev --log-level debug --dir aws-cluster/cluster-dev create cluster
```

Reply:
sorry was responding to above yes those files exist ie  `/assets/tls/etcd-metrics-ca-bundle.crt`